src/mirror-client-finder/directory-finder.ts

```diff
@@ -20,6 +21,11 @@ export async function findClientDir(client: InstallableClient, desiredVersionRan
     );
     const clientVersionedDir = clientBaseDir + clientMatchedVersion;
 
+    if (client === Inputs.CRC && getOS() === "macos" && clientMatchedVersion > "1.27.0") {
```

Comment:
This is a string compare. You have to use the semver library.

Reply:
otherwise it gets confused when you reach another decimal place:

```
Welcome to Node.js v14.17.0.
Type ".help" for more information.
> const version = "1.27.0"
undefined
> "1.100.0" > version
false
```